templates/utility.html

```diff
@@ -13,11 +12,22 @@
         {{error}}
     </div>
     {% endif %}
-
     {% set vars = {"readonly": "readonly" if status == "check" else "", "bank_index": 0} %}
-
     <div class="card">
-        <div class="card-header">
+    {% if kyc_info['level'] < '2' %}
```

Comment:
remove the oauth login/logout stuff from this template and into the layout.html (use a navbar: https://getbootstrap.com/docs/4.0/components/navbar/)

Reply:
This is a problem.

Cant figure a way to do this. We have a redirect_url going to utilities. The navbar doesn't take effect on the utility page.

Reply:
made this global